descriptions/css/properties/border-right.json

```diff
@@ -0,0 +1,9 @@
+{
+  "css": {
+    "properties": {
+      "border-right": {
+        "__short_description": "The <strong><code>border-right</code></strong> <a href='https://developer.mozilla.org/docs/Web/CSS'>CSS</a> property is a shorthand that sets the values of <a href='https://developer.mozilla.org/docs/Web/CSS/border-right-width'><code>border-right-width</code></a>, <a href='https://developer.mozilla.org/docs/Web/CSS/border-right-style'><code>border-right-style</code></a> and <a href='https://developer.mozilla.org/docs/Web/CSS/border-right-color'><code>border-right-color</code></a>."
+      }
```

Comment:
"The border-right CSS property is a shorthand that..." -> "The border-right shorthand CSS property..." ?